tables/da-dk-g28l.ctb

```diff
@@ -737,6 +567,440 @@ noback context _$d["nd"]!$l @1345-145
 noback context $d[]$l!$l @6
 noback context $d[]$l~ @6
 
+noback pass2 @f-235 @2357
```

Comment:
Would you mind adding some comment here explaining what these pass2 rules are all about?